diff -ur bc-1.05.old/bc/Makefile.am bc-1.05/bc/Makefile.am
--- bc-1.05.old/bc/Makefile.am	Sun Mar 29 07:40:52 1998
+++ bc-1.05/bc/Makefile.am	Tue Jun  1 10:53:54 1999
@@ -30,10 +30,10 @@
 
 install-data-local:
 	if grep -s "define BC_MATH_FILE" ../config.h; \
-	  then $(mkinstalldirs) $(libdir); \
-	  rm -f $(libdir)/libmath.b; \
-	  $(INSTALL_DATA) $(srcdir)/libmath.b $(libdir); \
-	  chmod 444 $(libdir)/libmath.b; \
+	  then $(mkinstalldirs) $(DESTDIR)$(libdir); \
+	  rm -f $(DESTDIR)$(libdir)/libmath.b; \
+	  $(INSTALL_DATA) $(srcdir)/libmath.b $(DESTDIR)$(libdir); \
+	  chmod 444 $(DESTDIR)$(libdir)/libmath.b; \
 	  else true; \
 	fi
 
